dataprofiler/tests/labelers/test_data_labelers.py

```diff
@@ -36,47 +36,79 @@ def test_train_method_exists(self):
         self.assertTrue(hasattr(dp, 'train_structured_labeler'))
 
         with mock.patch('dataprofiler.train_structured_labeler') as mock_obj:
-            dp.train_structured_labeler(None, None)
-            self.assertEqual(mock_obj.call_args, mock.call(None, None))
+            dp.train_structured_labeler(None, None, None)
+            self.assertEqual(mock_obj.call_args, mock.call(None, None, None))
```

Comment:
Shouldn't these 2 calls also just be passed `(None)` instead of `(None, None, None)`, so that it can match the input case below

Reply:
changed